backend/src/components/facilities.js

```diff
@@ -111,33 +115,36 @@ async function updateFacility(req, res) {
 }
 async function getFacilitiesForRenewal(req, res) {
   try {
-    const facilityIds = req.body?.facilityIds || []
+    const facilityIds = req.body?.facilityIds
     if (!facilityIds.length) return res.status(HttpStatus.OK).json([])
```

Comment:
- I don't think we need this here because we already check for empty list in two places before we reach this component. Line 119